File: Chapter01/R_Code/ScoreData.R
library(tidyverse)

model <- readRDS("./Models/model.rds")
boston_housing <- read_csv("./Data/BostonHousingInfo.csv")

model_data <- boston_housing[, c("crim","rm","tax","lstat")]
pred_medv <- predict(model, model_data)

final_output <- cbind(model_data, pred_medv)

File: Chapter01/Python_Code/ScoreData.py
import os
import pandas as pd
from sklearn.externals import joblib
from sklearn import linear_model

model = joblib.load("./Models/model")
boston_housing = pd.read_csv("./Data/BostonHousingInfo.csv")

model_data = boston_housing[["crim","rm","tax","lstat"]]

pred_medv = model.predict(model_data)

final_output = model_data
final_output["pred_mev"] = pred_medv